dashboard/src/components/ClassView/index.tsx

```diff
@@ -23,7 +23,10 @@ const RequiredRBACRoles: IRBACRole[] = [
 ];
 
 interface IClassViewProps {
-  classes: IClusterServiceClass[];
+  classes: {
```

Comment:
instead of repeating it, we can use `IServiceCatalogState["classes"]` as the type